webapp/utils.py

```diff
@@ -47,7 +47,10 @@ def parse_datetime(date):
     """Parse date from string in ISO format."""
     if date is None:
         return None
-    return dateutil_parser.parse(date)
+    ret = dateutil_parser.parse(date)
+    if not ret.tzinfo:
+        raise ValueError("Missing timezone")
```

Comment:
This is the error message which we send to a user together with 400 code. I think it should be more informative, something like:
`raise ValueError("Wrong date format (not ISO format): " + date)`